ext/drmgr/drmgr.h

```diff
@@ -1224,6 +1235,15 @@ DR_EXPORT
  */
 bool drmgr_unregister_signal_event(dr_signal_action_t (*func)(void *drcontext,
                                                               dr_siginfo_t *siginfo));
+
+DR_EXPORT
+/**
+ * Unregister a callback function for the signal event with user data supplied.
```

Comment:
Same thing: please add a link to the routine w/ the complete docs

Reply:
Done.